src/mlpack/core/dists/gamma_distribution.cpp

```diff
@@ -0,0 +1,76 @@
+/**
+ * @file gamma_distribution.cpp
+ * @author Yannis Mentekidis
+ *
+ * Implementation of the methods of GammaDistribution.
+ */
+#include "gamma_distribution.hpp"
+
+#include <boost/math/special_functions/digamma.hpp>
+#include <boost/math/special_functions/trigamma.hpp>
```

Comment:
Sounds good.  Either way, it is probably worth checking that the functions you are using are available in our earliest-supported version of Boost (1.49).